src/lib/utils/blend.js

```diff
@@ -0,0 +1,23 @@
+// get current blending settings
+export function getBlendMode(gl) {
```

Comment:
Good! We should check and restore the original blend mode at the start and end of picking!

However, when profiling, I found this list of getParameter calls takes an large proportion of execution time. Don't attempt to solve it now. This is just a note.